src/main/java/nl/b3p/tailormap/api/persistence/services/GeoService.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright (C) 2023 B3Partners B.V.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+package nl.b3p.tailormap.api.persistence.services;
+
+import java.time.Instant;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.Basic;
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+
+@Entity
+public class GeoService {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Basic(optional = false)
+    private String protocol;
+
+    /**
+     * The URL from which the capabilities of this service can be loaded and the URL to use for the
+     * service, except when the advertisedUrl should be used by explicit user request. TODO: never
+     * use URL in capabilities? TODO: explicitly specify relative URLs can be used? Or even if it
+     * should be automatically converted to a relative URL if the hostname/port matches our URL?
+     * TODO: what to do with parameters such as VERSION in the URL?
+     */
+    @Basic(optional = false)
+    private String url;
+
+    @Column(columnDefinition = "jsonb")
+    private Object authentication;
+
+    /**
+     * Original capabilities as received from the service. This can be used for capability
+     * information not already parsed in this entity, such as tiling information.
+     */
+    @Lob
+    @Basic(fetch = FetchType.LAZY)
+    private byte[] capabilities;
```

Comment:
Can we use an xml type for this? that would allow xpath use in the database; may need a custom build of PG, see https://www.postgresql.org/docs/current/datatype-xml.html

Reply:
Interesting idea, some services (which we don't support yet, but may in the future) such as ArcGIS REST, OGC GeoAPI, TileJSON etc. might not use XML, but we can have separate columns for that with the parsed caps. Already have a contentType there

Reply:
Lets not use xml type